data/flows/order_pizza.yml

```diff
@@ -0,0 +1,100 @@
+flows:
+  order_pizza:
+    name: order pizza
+    description: user asks for a pizza
+    nlu_trigger:
+    - intent:
+        name: inform_pizza
+        confidence_threshold: 0.5
+    steps:
+    - call: "fill_pizza_order"
+      next:
+        - if: slots.confirmation_order
+          then: "payment_options"
+        - else: "cancel_order"
+    - id: "payment_options"
+      collect: payment_option
+      next:
+        - if: slots.payment_option is 'card'
+          then: "use_card_details"
+        - if: slots.payment_option is 'membership points'
+          then: "use_membership_points"
+        - else: "cancel_order"
+    - id: "use_card_details"
+      collect: card_details
+      next:
+          - if: slots.card_details and slots.confirmation_order
+            then: "take_payment"
+          - else: "cancel_order"
+    - id: "take_payment"
+      action: utter_execute_payment
+      next: END
+    - id: "use_membership_points"
+      call: "use_membership_points"
+      next:
+        - if: slots.membership_points and slots.confirmation_order
+          then: "take_payment"
+        - else: "cancel_order"
+    - id: "cancel_order"
+      action: utter_cancel_order
+      next: END
+
+  fill_pizza_order:
+    name: fill pizza order
+    description: user is asked to fill out pizza order details
+    steps:
+      - collect: pizza
+      - collect: num_pizza
+      - collect: address
+      - collect: confirmation_order
+        reset_after_flow_ends: False
+
+  auth_user:
+    name: auth user
+    description: user is asked to authenticate before proceeding
+    if: False
+    steps:
+      - collect: token
+        next:
+            - if: slots.token
+              then: "successful_request"
+            - else: END
+      - id: "successful_request"
+        action: utter_token_success
+        next: END
```

Comment:
we have another authentication flow, can we reuse it here?

Reply:
Updated: https://github.com/RasaHQ/rasa-calm-demo/pull/30/commits/e7001fc9a9d963468e3845fd51afc96a36f0898b